students/aminavi/mailroom 2.py

```diff
@@ -0,0 +1,62 @@
+import sys
+
+donor_list = {'William': (653772.32 , 953772.32,100000),'Jeff', (8888777.22 ,877.33),'Paul', (663.23, 43.87 ,1.32),'Mark', ([)1663.23, 4300.87, 10432.00),'Nigel',(5000000 , 10)}
```

Comment:
Your donations are stored in tuples, not in lists. Wouldn't you get some errors on line 19 when you enter a donation amount?

Reply:
Well I guess your code is not completed yet, because there's definitely some syntax errors here in line 3, and your code in lines 17-21 is using index numbers, not names, so I assume you'll keep working on this and push completed commits...?